platform/mbed_retarget.cpp

```diff
@@ -913,7 +913,7 @@ extern "C" long PREFIX(_flen)(FILEHANDLE fh)
 }
 
 // Do not compile this code for TFM secure target
-#if !defined(COMPONENT_SPE) || !defined(TARGET_TFM)
```

Comment:
Why remove COMPONENT_SPE here?

Reply:
@Patater We do not have `COMPONENT_SPE` anymore or `FEATURE_SPE` as we'll not build secure side using mbed-cli. But in this case we need to identify non-secure and the current #if i snot doing that. I'll change the code to `#if defined(FEATURE_NSPE)` which is only defined for NSPE.